akka-docs/src/main/paradox/stream/operators/Source-or-Flow/throttle.md

```diff
@@ -14,6 +14,41 @@ Limit the throughput to a specific number of elements per time unit, or a specif
 Limit the throughput to a specific number of elements per time unit, or a specific total cost per time unit, where
 a function has to be provided to calculate the individual cost of each element.
 
+The throttle operator combines well with the @ref[`queue`](./../Source/queue.md) operator to adapt the speeds on both ends of the `queue`-`throttle` pair.
```

Comment:
queue is mentioned here, but it's not mandatory for throttle so I wonder if it's just adding confusion, or did you have something more in mind for when queue-throttle should be used rather than only throttle?

Reply:
The [`queue.md` page](https://doc.akka.io/docs/akka/current/stream/operators/Source/queue.html) has a cross-reference to `throttle.md` so I want readers to enter an infinite loop.

I'm trying to make it clear that `throttle` has no buffering in it but if you need it, then use `queue, it's a common pattern.